source/common/grpc/async_client_impl.h

```diff
@@ -158,12 +158,13 @@ class AsyncStreamImpl : public AsyncStream<RequestType>,
       streamError(Status::GrpcStatus::Internal);
       return;
     }
+    const std::string grpc_message = Common::getGrpcMessage(*trailers);
```

Comment:
Sorry I didn't notice this till just now. Can we do this inside the below if check? Then we can save the string copies in the common case. :)